ONSdigital/ras-frontstage
# RAS-1400 Extend party enrolment to include business and survey details

# What and why?
This PR updates the survey_list build process, instead of using redis and calling to party/survey multiple times it is provided on the first call. It also adds a call to a new endpoint specially for checking a respondent is enrolled 
# How to test?
Deploy this card as well https://github.com/ONSdigital/ras-party/pull/449. Add some enrolment data to your account, acceptance tests could be used, but you might want to add another IAC as well. If it logs in you are good. Also check you can access a eq, download and upload a seft
# Jira
